src/IW/Db/Issue.hs

```diff
@@ -14,32 +14,35 @@ import IW.Core.Issue (Issue (..))
 import IW.Core.Id (Id (..))
 import IW.Db.Functions (WithDb, asSingleRow, executeMany, query, queryRaw)
 
+
 -- | Returns all issues in the database
 getIssues :: (WithDb env m) => m [Issue]
 getIssues = queryRaw [sql|
-    SELECT id, number, title, body, repo_owner, repo_name, labels
+    SELECT id, repo_owner, repo_name, number, title, body, labels
     FROM issues
 |]
 
 -- | Returns a single issue with the corresponding ID
 getIssueById :: (WithDb env m, WithError m) => Id Issue -> m Issue
 getIssueById issueId = asSingleRow $ query [sql|
```

Comment:
I think for now you can remove `getIssueById` function. Just to have less code to test and worry about. We can easily write it back again when we need it.